### PR TITLE
1.19

### PR DESCRIPTION
**Pull request by fantahund**
Update 1.19 (FullBright broken... thx Mojang)

### CODE REVIEW

**Comment by ate47** [comment]
Any idea for the fullbright bug?

**Comment by fantahund** [comment]
I do not know yet had no time to look at me but I can tomorrow or Wednesday times look there I have time we have this mod only needed because we use it on our server to condole Xray users. ^^

**Comment by ate47** [comment]
Thanks for the update, I will check if I can fix the fullbright and upload it asap :)

edit: [CurseForge file](https://www.curseforge.com/minecraft/mc-mods/xray-1-13-rift-modloader/files/3832345)

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -21,9 +21,9 @@
   "mixins": ["xray.sodium.mixins.json", "xray.vanilla.mixins.json"],
 
   "depends": {
-    "fabricloader": ">=0.11.3",
+    "fabricloader": ">=0.14.7",
     "fabric": "*",
-    "minecraft": "1.18.x"
+    "minecraft": "1.19"
```
Comment (ate47) [inline src/main/resources/fabric.mod.json:26]: Don't forget the `.x` next time ;)

Comment (ate47) [reply src/main/resources/fabric.mod.json:26]: I fixed it in 1155787803d850caaefd71caffbf2079b3e9bb97

**Pull request merged**